src/test/components/FlameGraph.test.js

```diff
@@ -71,18 +72,32 @@ describe('FlameGraph', function() {
   });
 
   it('shows a tooltip when hovering', () => {
-    const { getTooltip, moveMouse } = setupFlameGraph();
+    const { getTooltip, moveMouse, findFillTextPosition } = setupFlameGraph();
     expect(getTooltip()).toBe(null);
-    moveMouse(GRAPH_WIDTH * 0.5, GRAPH_HEIGHT - 3);
+    moveMouse(findFillTextPosition('A'));
     expect(getTooltip()).toBeTruthy();
   });
 
   it('has a tooltip that matches the snapshot', () => {
-    const { getTooltip, moveMouse } = setupFlameGraph();
-    moveMouse(GRAPH_WIDTH * 0.5, GRAPH_HEIGHT - 3);
+    const { getTooltip, moveMouse, findFillTextPosition } = setupFlameGraph();
+    moveMouse(findFillTextPosition('A'));
     expect(getTooltip()).toMatchSnapshot();
   });
 
+  it('shows a tooltip with the resource information', () => {
+    const { getTooltip, moveMouse, findFillTextPosition } = setupFlameGraph();
+    moveMouse(findFillTextPosition('J'));
+    const tooltip = ensureExists(getTooltip());
+
+    // First, a targeted text.
```

Comment:
nit: should it be "targeted test" btw? not sure if this is the intended comment, but work for me either way :)

Reply:
Ooops yep :)